### PR TITLE
Add data classes follow-up article

### PR DESCRIPTION
**Pull request by balapriyac**

### CODE REVIEW

**Comment by balapriyac** [comment]
Preview link: https://642995f7b233891d69779e22--ephemeral-narwhal-3ac3b2.netlify.app/blog/more-on-python-data-classes/

**Comment by balapriyac** [comment]
@jalletto the data classes follow-up article is ready to review.
Preview link: https://642a880c955a573dde7a7275--ephemeral-narwhal-3ac3b2.netlify.app/blog/more-on-python-data-classes/

**Review by jalletto — APPROVED** [review]

**Comment by jalletto** [comment]
@balapriyac Really great work on this! I learned a lot. Your asides are always really well timed and helpful.

File: blog/_posts/2023-04-21-more-on-python-data-classes.md [hunk 1]
```diff
@@ -0,0 +1,717 @@
...
+author: Bala Priya C
+
+internal-links:
+ - Python
+ - Python Classes
+---
+
+Since their introduction in Python 3.7, data classes have emerged as a popular choice for Python classes that store data. In a [previous tutorial](/blog/python-data-classes), we talked about what data classes are and some of their features, including out-of-the-box support for object comparison, type hints, and default values of fields. In this follow-up tutorial, we'll continue to explore some more features of Python data classes.
+
+Over the next few minutes, we'll take a closer look at setting default values with `default_factory`, initializing new fields from pre-existing fields with `__post_init__`, and much more. We'll also discuss the *improved* support for `__slots__` in data classes since Python 3.10.
```
[the remark this answers is not in the document]
Comment (balapriyac) [reply blog/_posts/2023-04-21-more-on-python-data-classes.md:15]: > "Over the next few minutes" sounds weird to me. The article is a 16 minute read also.

Agreed, I'll remove that. It's not a few minutes. Should take about 20 minutes to get through the whole article.

File: blog/_posts/2023-04-21-more-on-python-data-classes.md [hunk 2]
```diff
@@ -0,0 +1,717 @@
...
+
+~~~{ caption="Output"}
+Joy Smith's tuition: 10000
+Julia Doe's tuition: 27000
+Jake Langdon's tuition: 28000
+Jane Lee's tuition: 30000
+~~~
+
+<div class="notice--big--primary">
+### Setting `order=True` Facilitates Comparison. But How?
```
[the remark this answers is not in the document]
Comment (balapriyac) [reply blog/_posts/2023-04-21-more-on-python-data-classes.md:386]: > @balapriyac Really great work on this! I learned a lot. Your asides are always really well timed and helpful.

Thank you very much for the review and feedback @jalletto. I removed "over the next few minutes.." from the intro.

**Comment by balapriyac** [comment]
> @balapriyac Really great work on this! I learned a lot. Your asides are always really well timed and helpful.

Thank you very much for the review and feedback @jalletto. I removed "over the next few minutes.." from the intro.

**Pull request merged**